src/main/java/frc/robot/RobotContainer.java

```diff
@@ -36,19 +40,22 @@ public class RobotContainer {
   private final Climber s_Climber = new Climber();
 
   // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
-      new CommandXboxController(Constants.kDriverControllerPort);
+  private CommandXboxController m_driverController = new CommandXboxController(Constants.kDriverControllerPort);
 
-  private final CommandXboxController m_SecondaryController =
-      new CommandXboxController(Constants.kSecondaryControllerPort);
+  private CommandXboxController m_SecondaryController = new CommandXboxController(
+      Constants.kSecondaryControllerPort);
 
   // Chooser for autonomous routines
   SendableChooser<Command> m_chooser = new SendableChooser<>();
 
   // All auto routines
   private final AutoRoutines m_autoRoutines = new AutoRoutines(s_DriveTrain, s_IntakeShooter);
 
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
+  boolean controllersInverted;
```

Comment:
Controller inversion should be done in driver station, not here

Reply:
Its to handle the issue quickly if it happens, Driverstation takes time.
If you really think it is a bad idea, Ill remove it.